From: Dave Plater <davejplater@gmail.com>
Date: 2019-03-20
Subject: change in boost_headers
References: release 0.37.3
Upstream: https://sourceforge.net/p/guitarix/patches/10/

boost/core/noncopyable.hpp needs to be specifically included.

Index: src/headers/engine.h
===================================================================
--- src/headers/engine.h.orig	2020-01-14 09:47:57.000000000 +0200
+++ src/headers/engine.h	2020-01-16 11:42:17.223018118 +0200
@@ -40,6 +40,7 @@
 #include <boost/format.hpp>
 #include <boost/noncopyable.hpp>
 #include <boost/thread/mutex.hpp>
+#include <boost/core/noncopyable.hpp>
 #include <glibmm/i18n.h>     // NOLINT
 #include <glibmm/optioncontext.h>   // NOLINT
 #include <glibmm/dispatcher.h>
